src/components/Badge/Badge.scss

```diff
@@ -1,17 +1,16 @@
 @import '../../styles/common';
-$horizontal-padding: spacing(tight);
-$vertical-padding: calc(#{spacing(extra-tight)} - #{rem(2px)});
-
-$pip-size: spacing(tight);
-$pip-spacing: calc(#{$pip-size} / 2);
 
 .Badge {
+  --p-horizontal-padding: var(--p-space-2);
+  --p-vertical-padding: calc(var(--p-space-1) - #{rem(2px)});
+  --p-pip-size: var(--p-space-2);
+  --p-pip-spacing: calc(var(--p-pip-size) / 2);
```

Comment:
The value for these custom properties seems quite low. I think it would be safe to remove them. Can we also change `calc(var(--p-pip-size) / 2)` to `--p-space-1`.